pkg/chains/formats/all/all.go

```diff
@@ -17,5 +17,6 @@ package all
 import (
 	_ "github.com/tektoncd/chains/pkg/chains/formats/simple"
 	_ "github.com/tektoncd/chains/pkg/chains/formats/slsa/v1"
+	_ "github.com/tektoncd/chains/pkg/chains/formats/slsa/v2"
```

Comment:
I'd hold off on wiring this up just yet - there's probably more work we need to do for pipelineruns (we can still include this for tests in the meantime).

Reply:
If you want this for e2e tests, we can place this behind a build flag.

Reply:
Yes, I agree. There might be more work for pipelineruns. Hiding it behind a build flag works since I do want e2e tests for this.

Reply:
@wlynch how should I create a build flag and hide this feature behind it? Did you mean some type of a configmap field like we have feature flags?

Reply:
One more thought. Since [TEP0122](https://github.com/tektoncd/community/pull/820) only covers taskruns, and `slsa/v2` is currently only activated for taskruns, why do we need to wait for Pipelineruns? Can we not make it available for taskrun provenances?

Reply:
I like the `v2alpha1` option because it will allow the users to test it if they like and give us feedback.

Reply:
I implemented `v2alpha1` and also surfaced it in the docs and left notes. Let me know if this is ok.